sprint-2/src/Types.ts

```diff
@@ -0,0 +1,6 @@
+export type Person = {
+    readonly id: number;
+    name: string;
+    bio: string;
+    [key: string]: string|number;
```

Comment:
Porque você precisa de um atributo genérico? É melhor manter atributos fixos

Reply:
Removi o atributo em questão.

Reply:
Pelo o que eu acabei de testar esse atributo seria para poder usar um loop `for in` no objeto, mas de fato eu não estava usando nesse código em específico